File: Java101Pratikler/AtmProject/New.java
import java.util.Scanner;

public class New {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int num1, num2;
            boolean result = true, mistake=false;

        System.out.print("İlk sayıyı girin : ");
        num1 = input.nextInt();

        System.out.print("İkinci sayıyı girin : ");
        num2 = input.nextInt();

        if (num2 > num1){
            System.out.println(result);
        }else if (num2 == num1){
            System.out.println("-1");
        }else {
            System.out.println(mistake);
        }
    }
}

File: Java101Homeworks/SayiTahmin/New.kt
import java.util.*

object New {
    @JvmStatic
    fun main(args: Array<String>) {
        val scanner = Scanner(System.`in`)
        println("ilk Sayı")
        val n1 = scanner.nextInt()
        println("İkinci ")
        val n2 = scanner.nextInt()
        var ebob = 1
        for (i in 1..n1) {
            if (n1 % i == 0 && n2 % i == 0) {
                ebob = i
            }
        }
        println(ebob)
    }
}